pages/sign-up.js

```diff
@@ -54,46 +77,38 @@ const SignUp = () => {
 
                         <Input label="Confirm Password" name="confirmPassword" type="password" />
 
-                        <div className="flex items-center mt-8 mb-3">
-                          <input
-                            id="remember-me"
-                            name="remember-me"
-                            type="checkbox"
-                            placeholder="Your password"
-                            className="w-4 h-4 text-amcovad-primary-400 border-gray-200 rounded focus:ring-amcovad-primary-300 "
+                        <div className=" mt-8 mb-3">
+                          <CheckBoxGroup name="acceptMulti" options={output} />
+                          <Checkbox
```

Comment:
Let's convert this to CheckboxGroup. Let's add value to the props here too.

Reply:
thought the value is already in the `options={output}` test array?

Reply:
Not sure I understand your statement above. I was referring to line 82 where you used Checkbox to declare the "acceptTerms".

```js
<Checkbox
    name="acceptTerms"
...
```

I am saying that we should convert the Checkbox you used here to CheckboxGroup. It has nothing to do with the `options={output}` test array as we are removing that one too?